src/components/beers.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import axios from "axios";
+import Header from "./header";
+import '../App.css';
+
+const Beers = () => {
+    const [beerListResults, setBeerListResults] = useState([]);
+  
+        useEffect(() => {
+            axios.get(`https://ih-beers-api2.herokuapp.com/beers`)
+            .then(beerResults => setBeerListResults(beerResults.data),
+            console.log(beerListResults))
+            .catch(error => console.log(`${error}`))
+        }, [])
+
+  return (
+   <div >
+    <Header /> 
+    {beerListResults.length ? 
+        beerListResults.map((beer, index) => {
+        return(
+            
+            <div className="card" key={index}>
```

Comment:
It's better practice to use an id for the `key` attribute when you have the option. 
```
<div className="card" key={beer._id}>